components/Chips/src/MDCChipView.h

```diff
@@ -161,6 +161,27 @@
 - (void)setBackgroundColor:(nullable UIColor *)backgroundColor forState:(UIControlState)state
     UI_APPEARANCE_SELECTOR;
 
+/*
+ A color used as the chip's color for @c state.
```

Comment:
I wonder if "background overlay color" might be a more accurate/descriptive term?

Reply:
Hmmmm I don't really wanna change the existing API since we have to deprecate it. But we could plan this in future.

Reply:
okay I could change the name.